content/docs/for-developers/sending-email/building-an-x-smtpapi-header.md

```diff
@@ -41,22 +54,12 @@ SMTP works by passing a JSON string with as many SMTP objects as you want to Sen
 }
 ```
 
-### Limitations
-
-- Ensure that you are sending just ONE header. Failure to do so, may cause unwanted consequences with inconsistent behavior. 
-- There is a hard limit of 10,000 addresses in a multiple recipient email. However, the best practice is to split up large jobs to around 1,000 recipients, which allows better processing load distribution. If you have a large number of additional substitutions or sections in the headers, it is best to split the send into even smaller groups.
-- When using the X-SMTPAPI to send to multiple recipients, you cannot use the standard SMTP protocol's "TO" field to send to multiple recipients because doing so can generate duplicate messages to the addresses listed in both. For more information, see [RFC 5321](https://tools.ietf.org/html/rfc5321).
-- Ensure that the header is limited to a maximum total line length of 1,000 characters. Failure to do this can cause intermediate MTAs to split the header on non-space boundaries, which causes inserted spaces in the final email. If your email is going through another MTA before reaching SendGrid, it is likely to have an even lower setting for maximum header length and may truncate the header.
-- When using the API, if our system encounters a parsing error, the message will be bounced to the address specified in the MAIL FROM portion of the SMTP session. The MAIL FROM address is re-written when we send the email out for final delivery, so it is safe to set this to an address that can receive the bounces so that you will be alerted to any errors.
-- When sending Unicode characters via the SMTP API, you should escape these characters using the `\u` escape character. When you do this, Unicode characters like `á` becomes `\u00E1`.
-
-## Customizing your send (filters)
+You can customize the emails you send via SMTP by using different settings (also referred to as filters).
```

Comment:
Should we not link to the filter settings direct here?

Reply:
Good call. I'll get a link in there.

Reply:
lgtm